workflow/config/config.yaml

```diff
@@ -12,69 +21,108 @@ train_seeds:
   - 7
   - 8
   - 9
+
+# Seeds used when sampling molecules from the trained models.
+# The amount of 'sample_seeds' parameter specifies how many times the 'sample_molecules_RNN' step is executed,
+# each time using the same trained model but with different random seed values.
 sample_seeds:
   - 0
+
+# Specifies by how many times the input data is augmented (or enumerated) before training.
+# This utilizes the fact that a single molecule can have multiple SMILES representation.
+# For example, a value of 0 means no augmentation, while 100 means each molecule can have
+# up to 100 different representations in the training set.
 enum_factors:
   - 0
   - 10
   - 30
   - 50
   - 100
+
+#  Limits the maximum number of input SMILES to be processed. 0 means there's no limit.
 max_input_smiles: 0
+
+# Parameters that define the neural network model and training process.
 model_params:
-  rnn_type: LSTM
-  embedding_size: 128
-  hidden_size: 1024
-  n_layers: 3
-  dropout: 0
-  batch_size: 64
-  learning_rate: 0.001
-  max_epochs: 99999
-  patience: 50000
-  log_every_steps: 100
-  log_every_epochs: 1
-  sample_mols: 10000000
+  rnn_type: LSTM # Type of Recurrent Neural Network (RNN) to use.
```

Comment:
Agree - similar for representation (SMILES vs SELFIES being the two options)

Reply:
@skinnider - since you raise this as an example - I'm fairly certain that running the workflow on SELFIES will not work at all and will open up a can of worms (I think we realized early on that we needed to handle selfies as a special case in certain steps, and decided to not handle it). Perhaps we remove all mentions of SELFIES? (and add an issue to look into them if we need to?)

Reply:
Yes, that would be very reasonable since I think we will never use SELFIES going forward.

Reply:
I am running a workflow on argo with SELFIES. It's on the training step right now. I'll let you know if anything breaks along the way.